src/main.js

```diff
@@ -98,15 +98,24 @@ function makeCoverButton() {
   homeView.classList.add('hidden');
   saveCoverButton.classList.add('hidden');
   showNewRandomCoverButton.classList.add('hidden');
+  savedView.classList.add('hidden');
 
   formView.classList.remove('hidden')
   homeButton.classList.remove('hidden');
 }
 
+function saveCover() {
+  if (!savedCovers.includes(currentCover)) {
+    savedCovers.push(currentCover);
+    console.log(savedCovers);
```

Comment:
please remove all console logs! These should never make it into production code 😄

Reply:
Oops! Done!